apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/WhenToSendCard.tsx

```diff
@@ -209,7 +238,14 @@ export default function WhenToSendCard({
             </Button>
           </div>
 
-          <div className="ml-2 flex items-center space-x-1 px-4 pb-4">
+          <div className="ml-2 flex items-center space-x-1 px-4 pb-4"></div>
+          <AdvancedOptionToggle
```

Comment:
<img width="686" alt="Screenshot 2024-01-29 at 09 13 12" src="https://github.com/formbricks/formbricks/assets/675065/ed363aef-2051-4f82-bd41-cfa438e1e92b">

please make the blue dot black to fit better with the other styles 😊